spec/cypress/cypress_helper.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+# This is loaded once before the first command is executed
+
+begin
+  require 'database_cleaner-active_record'
+rescue LoadError => e
+  puts e.message
+  begin
+    require 'database_cleaner'
+  rescue LoadError => e
+    puts e.message
+  end
+end
+
+begin
+  require 'factory_bot_rails'
+rescue LoadError => e
+  puts e.message
+  begin
+    require 'factory_girl_rails'
+  rescue LoadError => e
+    puts e.message
+  end
+end
+
+require 'cypress_on_rails/smart_factory_wrapper'
+
+factory = CypressOnRails::SimpleRailsFactory
+factory = FactoryBot if defined?(FactoryBot)
+factory = FactoryGirl if defined?(FactoryGirl)
```

Comment:
Where does this FactoryGirl come from?
it has been deprecated aeons ago

Reply:
I was wondering about this as well. That's boilerplate generated by https://github.com/shakacode/cypress-on-rails.
We need the gem for integrating Rails with Cypress.

Reply:
In fairness, they only seem to use `FactoryGirl` for compatibility with legacy applications.